packages/flutter_tools/lib/src/ios/mac.dart

```
@@ -55,6 +55,7 @@ class Xcode {
           else
```

cc @johnmccutchan for changes to the tool to avoid empty catch blocks